great-idea/index.html

```diff
@@ -16,8 +18,122 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+<div class="massContainer">
+
+<div class="navContainer">
+  
+  <div class="navBar">
+
+  <a href="services.html"> Service </a>
```

Comment:
Nice job putting in these links - you'll want to use "#" for hrefs without the page existing yet so that if a user clicks them, it won't error looking for an html file that doesn't exist.